docs/VTPM.md

```diff
@@ -41,3 +41,70 @@ sequenceDiagram
 ```
 
 For a working example of this operation, please check `/pkg/vtpm/vtpm-attest/`. Please note that if a hardware TPM is not available to EVE, these security guarantees are nulled and the virtual TPM state is stored unencrypted.
+
+## Example Usecase : Azure IoT Edge
+
+Using the VTPM, we can use the TPM based Device Provisioning service from Azure Portal. This guide walks you through a "Individual Enrollment" setup based on Ubuntu 22.04 and Azure IoT Edge 1.5.x .
+
+### Setup
+
+1- We start by creating the necessary keys in the TPM. The two required keys are the Endorsement Key (EK) and the Storage Root Key (SRK). The default configuration expects to find them in TPM indexes `0x81010001` and `0x81000001`, respectively. If a different index is used, it must be set by changing `auth_key_index` in the IoT Edge configuration file.
```

Comment:
I'm just curious: is it an intentional usage of a not-markdown list here? )

Reply:
Yetus complains about it not being ordered, even though it is, I think it is because of the code sections in between. 
```
/home/shah/shah-dev/eve/docs/VTPM.md:109:1:markdownlint:MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 6; Style: 1/1/1]
/home/shah/shah-dev/eve/docs/VTPM.md:64:1:markdownlint:MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/1/1]
/home/shah/shah-dev/eve/docs/VTPM.md:74:1:markdownlint:MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 3; Style: 1/2/3]
/home/shah/shah-dev/eve/docs/VTPM.md:76:1:markdownlint:MD029/ol-prefix Ordered list item prefix [Expected: 2; Actual: 4; Style: 1/2/3]
/home/shah/shah-dev/eve/docs/VTPM.md:90:1:markdownlint:MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 5; Style: 1/1/1]
````

Reply:
Yep... I always have the same problem =( Ok, I think I'll borrow your trick.